test/built-ins/Proxy/ownKeys/extensible-return-trap-result.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.12
+description: >
+    If target is extensible, return the trap result.
+info: >
+    [[OwnPropertyKeys]] ( )
+
+    ...
+    22. If extensibleTarget is true, return trapResult.
+---*/
+
+var target = {};
+
+Object.defineProperty(target, "foo", {
+    configurable: false,
+    enumerable: true,
+    value: true
+});
+
+var p = new Proxy(target, {
+    ownKeys: function() {
+        return ["foo", "bar"];
+    }
+});
+
+var keys = Object.keys(p);
+
+assert.sameValue(keys[0], "foo");
+assert.sameValue(keys[1], "bar");
```

Comment:
`keys.length` is 1, so this assertion isn't correct. Change `Object.getOwnPropertyNames` to `Object.keys`? Probably also in other tests ownKeys trap?